Shared/UI/Face Detection/ImageCaptureViewController.swift

```diff
@@ -42,7 +42,7 @@ class ImageCaptureViewController: ViewController, AVCaptureVideoDataOutputSample
                 self.configureCaptureSession()
                 self.session.startRunning()
             }
-        }.add(to: self.taskPool)
+        }.add(to: self.autocancelTaskPool)
```

Comment:
Here's an example where adding a task to the task pool doesn't have any effect. Even after the view controller is dismissed this task will continue to run.

We can revisit these later and add cancellation checks.